docker/experian-crosscore-auth/openapi.yml

```diff
@@ -0,0 +1,114 @@
+swagger: '2.0'
+info:
+  title: CrossCore API
+  version: '2.21.0'
+host: env-region-api.experian.com
+basePath: /decisionanalytics/crosscore/{routing_id}/services/v0/applications
+schemes:
+  - https
+paths:
+  /api/v1/healthcheck:
+    get:
+      summary: Healthcheck
+      responses:
+        "200":
+          description: Success
+          schema:
+            $ref: "#/definitions/GenericResponse"
+        "401":
+          description: Error
+  /oauth2/experianone/v1/token:
+    post:
+      summary: Oauth2 token
+      parameters:
+        - in: header
+          name: Content-Type
+          type: string
+        - in: header
+          name: Accept
+          type: string
+          description: 'Accepts mime type application/json'
+        - in: header
+          name: X-Correlation-Id
+          type: string
+          description: 'Auto generated UUID'
+        - in: header
+          name: X-User-Domain
+          type: string
+          description: 'Unique domain for client'
+        - in: body
+          name: body
+          description: CrossCore Request
+          required: true
+          schema:
+            $ref: '#/definitions/TokenRequest'
```

Comment:
This isn't how you define bodies in OpenAPI and I don't think it will work (certainly, I can't make it work locally)